pkg/internal/third_party/go-json-experiment/README.md

```diff
@@ -0,0 +1,5 @@
+Forked from: https://github.com/go-json-experiment/json
+Commit Hash: df063377ba2ad1341f6d446add098f612d402ec6
```

Comment:
You should update that hash, I don't think you did (and it's hard to verify, that's why we need to get it right).

Reply:
nice catch

Reply:
😎